README.md

```diff
@@ -0,0 +1,26 @@
+<h1>AdoptaDog</h1>
```

Comment:
for markdown there is another styling language all together. https://guides.github.com/features/mastering-markdown/